plugins/sectionManager.js

```diff
@@ -0,0 +1,65 @@
+
+const fs = require('fs');
+const path = require('path');
+const marked = require('marked');
+const config = require('../getConfig').get();
+
+module.exports = {
+  
+  listSections: [],
+  
+  process(filename, data) {
+    let fileString = data.contents.toString();
+      
+    if (match = fileString.match(/(\[section=)[a-zA-Z0-9]+\]/g)) {
```

Comment:
Please properly declare that `match` variable beforehand, and then test it. This current code is confusing (my 1st thought was that the assignment operator was used instead of an equality one)